test/test.env

```diff
@@ -15,3 +15,9 @@ LOGIN_ATTEMPT_CAP=2
 DISABLE_REQUEST_LOGGING=true
 NODE_TEST_MODE=true
 NODE_ENV=test
+GOCARDLESS_TEST_CLIENT_ID=cjchuyewhdj23d8279dh29test
+GOCARDLESS_TEST_CLIENT_SECRET=nco2i3914yu1icbckjbeco3test
+GOCARDLESS_LIVE_CLIENT_ID=acsfhuyewbvd3d8279dh29live
+GOCARDLESS_LIVE_CLIENT_SECRET=saq2i3914yu1icbckjbeco3live
```

Comment:
Nope, these are some values added by me, probably should have been easier for `test-client-id`. Sorry about the confusion

Reply:
Cool, could you change that to something like:
```
GOCARDLESS_TEST_CLIENT_ID=fake-client-id-test
GOCARDLESS_TEST_CLIENT_SECRET=fake-client-secret-test
GOCARDLESS_LIVE_CLIENT_ID=fake-client-id-live
GOCARDLESS_LIVE_CLIENT_SECRET=fake-client-secret-live
```
Thanks